firmware/application/apps/analog_audio_app.cpp

```diff
@@ -187,13 +187,20 @@ AnalogAudioView::AnalogAudioView(
         field_frequency.set_value(receiver_model.target_frequency() + offset);
     };
 
-    audio::output::start();
```

Comment:
190  :  Is it OK to remove from here, that audio::output::start ();  ?

Reply:
It should be because `on_modulation_changed` will do everything it needs to do to set up the audio for the given modulation.

Reply:
@kallanreed   We may need to rename the Audio app now though, because it's not outputting any sounds anymore...  :-)    (I haven't put this line back in to test if that's the reason yet)

Reply:
Adding this line back in restores the audio output.

Reply:
😅 how does update_modulation work then.

Reply:
I’ll put it back then.